scripts/patch-realsense-ubuntu-jammy-rpi4.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+#Break execution on any error received
+set -e
+
+# Note: In contrast to many of the other patch-* variants in this scripts folder
+# this only targets a single Ubuntu release and a single platform
+#    Ubuntu 22.04 jammy
+#    RaspberryPi4
+# Arguably, I think this script is much cleaner than trying to support a wide
+# range of Ubuntu releases and a wide range of board. I think this is clear
+# enough that it would be a good launching point for trying to adapt to your
+# distribution and your board.
+
+# Some background on the patched kernel modules. I had to go hunting for this information in a bunch
```

Comment:
Great work here.  I too had to hunt for this, and would love to share more in the documentation on installation.